test/DbCleaner.ts

```diff
@@ -1,15 +1,26 @@
-import type { PrismaClient } from '@prisma/client'
+import { sql } from 'drizzle-orm'
+import type { PostgresJsDatabase } from 'drizzle-orm/postgres-js'
 
 export enum DB_MODEL {
   User = 'user',
+  Post = 'post',
 }
 
-export async function cleanTables(prisma: PrismaClient, modelNames: readonly DB_MODEL[]) {
-  const delegates = modelNames.map<{ deleteMany: () => Promise<unknown> }>(
-    (modelName) => prisma[modelName],
-  )
+export async function cleanTables(drizzle: PostgresJsDatabase, modelNames: readonly DB_MODEL[]) {
+  for (const model of modelNames) {
+    const tableName = model.valueOf()
+    const schemaName = getSchemaForModel(model)
+    await drizzle.execute(sql.raw(`delete from "${schemaName}".${tableName}`))
+  }
+}
 
-  for (const delegate of delegates) {
-    await delegate.deleteMany()
+function getSchemaForModel(model: DB_MODEL): string {
+  switch (model) {
+    case DB_MODEL.User:
```

Comment:
is it possible to avoid manual enumeration here? does drizzle provide something we can iterate over?

Reply:
Changed in https://github.com/lokalise/node-service-template/pull/917/commits/a70df622b42f02bec5bffe36061992c18f86523f. The right schema is automatically used in this way, but I changed a bit `DB_MODEL` to map the table name and the table entity, which is needed to perform deletion in this way. I tried to keep the usage of `DB_MODEL` the same as it was before though so that it's easier to migrate